Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ModelOverrider.cs

```diff
@@ -105,6 +108,19 @@ void GetAssetPathFromCommandLine()
                 {
                     m_BehaviorNameOverrideDirectory = args[i + 1].Trim();
                 }
+                else if (args[i] == k_CommandLineModelOverrideExtensionFlag && i < args.Length-1)
```

Comment:
I added support to allow overriding with .onnx files, but later found out that that we need to make some other changes in the import code to support this. Waiting to hear back from the Barracuda team the best way to handle this.